### PR TITLE
Theming of Cachet is much simpler with more color settings

### PR DESCRIPTION
**Pull request by jbrooksuk**
It aims to:

1. Make all status pages look unique.
2. Give way more flexibility in the colours.
3. Remove "hard set" colours from `_status-page.scss` since they can default in the `ThemeComposer`.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
CC @cachethq/owners let me know what you think.

File: resources/views/index.blade.php [hunk 1]
```diff
@@ -1,85 +1,85 @@
 @extends('layout.master')
 
 @section('content')
-    @if($subscribersEnabled)
-    <div class="pull-right">
-        <p><a class="btn btn-success btn-outline" href="{{ route('subscribe.subscribe') }}">{{ trans('cachet.subscriber.button') }}</a></p>
-    </div>
-    @endif
+@if($subscribersEnabled)
```
Comment (GrahamCampbell) [inline resources/views/index.blade.php]: yeh

Comment (jbrooksuk) [inline resources/views/index.blade.php]: I hate the indents.

Comment (GrahamCampbell) [inline resources/views/index.blade.php]: refering to not being snake_case

Comment (jbrooksuk) [inline resources/views/index.blade.php]: Oh yeah... I'm not sure why that's like it actually, but I'll fix it on this branch.

**Comment by joecohens** [comment]
:+1:

**Comment by jbrooksuk** [comment]
Any changes required in the theme to accommodate the theme variables? Did you try setting custom colours?

File: app/Composers/AppComposer.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ public function compose(View $view)
         $mailFrom = env('MAIL_NAME', false);
 
         $withData = [
-            'subscribersEnabled' => $isEnabled && $mailAddress && $mailFrom,
+            'subscribers_enabled' => $isEnabled && $mailAddress && $mailFrom,
         ];
 
         $view->with($withData);
```
Comment (GrahamCampbell) [inline app/Composers/AppComposer.php]: should just be `view->withSubscribersEnabled`

**Comment by GrahamCampbell** [comment]
Needs rebasing.

**Comment by jbrooksuk** [comment]
@GrahamCampbell rebased, don't merge yet.

@joecohens could you sort that panel issue please? I can't find a good solution.

**Comment by GrahamCampbell** [comment]
Might be an idea to squash this to one commit before joe commits, just to keep this sane, especially if another rebase is needed later.

**Comment by jbrooksuk** [comment]
On it.

**Comment by jbrooksuk** [comment]
Done.

**Comment by joecohens** [comment]
On it :)

**Comment by jbrooksuk** [comment]
Do we want this to be highlighted too?

https://cloudup.com/cpP4NOZmB9I

**Comment by joecohens** [comment]
Yeah we do! I've made the changes on the last commit to fill the whole incident area, the status icon and the little arrow :)

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**